README.rst

```diff
@@ -153,7 +155,9 @@ configuration value::
 
 
 If the user sets ``DEBUG`` with a bad value, they get a helpful error message
-with the documentation for the configuration option and the ``ConfigManager``::
+with the documentation for the configuration option and the ``ConfigManager``:
+
+.. highlight:: none
```

Comment:
I'd use `.. code-block:: [language]` exclusively through the changes. This snippet is shell, so this should be:

```
.. code-block:: shell

   $ DEBUG=foo python myprogram.py
   <traceback>
```